setup.py

```diff
@@ -71,7 +67,8 @@
 setuptools_kwargs = {
     'zip_safe': False,
     'scripts': [],
-    'include_package_data': True
+    'include_package_data': True,
+    'python_requires': '>=3.5, <4'
```

Comment:
@wholmgren I remembered that this is the more "proper" way to call this out. This way, `pip install -e .` won't work if the Python version is wrong.

Reply:
There's an even shorter syntax for this: '~=3.5'. See https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Reply:
Good find!